mautrix_telegram/portal.py

```diff
@@ -2318,6 +2342,18 @@ async def _handle_telegram_document(
                 "image/": MessageType.IMAGE,
             }.get(info.mimetype[:6], MessageType.FILE),
         )
+        if attrs.is_audio:
+            content["org.matrix.msc1767.file"] = {
+                "url": content.url,
+                "name": content.body,
+                **(content.file.serialize() if content.file else {}),
+                **(content.info.serialize() if content.info else {}),
+            }
```

Comment:
Also just noticed that this happens before `content.file` and `content.url` are set, but the whole thing is unnecessary anyway so let's just get rid of it

```suggestion
```